lib/OrganizationsFieldArray/EditOrganizationCard.js

```diff
@@ -139,22 +145,48 @@ export default class EditOrganizationCard extends React.Component {
               renderTrigger={(props) => {
                 this.triggerButton = props.buttonRef;
 
+                let orgName = '';
```

Comment:
It'd be cleaner to just `const org = get(this.props.organization, 'org.name');`

Reply:
This has now been changed, along with other similar instances